prover/docs/01_gcp_vm.md

```diff
@@ -0,0 +1,140 @@
+# Creating a GCP VM
+
+In this section we will cover the creation of a VM suitable for prover development. We assume that you already have
+access to the GCP cluster.
+
+## Create a VM
+
+Open [Google cloud console](https://console.cloud.google.com/) and choose "Compute Engine".
+
+On the "Compute Engine" page choose the cluster suitable for creating VMs with GPU, and then click on "Create instance".
+
+We will need an GPU **L4** instance, so find the zone that is close to you geographically and has such instances. At the
```

Comment:
I'd mention that currently L4 is the cheapest GPU that has 24GB of VRAM - but you can pick more expensive one if you want